src/LogicTasks/Semantics/Decide.hs

```diff
@@ -93,8 +81,8 @@ description withDropdowns DecideInst{..} = do
     pure ()
   paragraph $ do
     translate $ do
-      english "Find all faulty truth values in the last column of the following truth table."
-      german "Finden Sie alle fehlerhaften Wahrheitswerte in der letzten Spalte der folgenden Wahrheitstafel."
+      english "Decide for each table row whether the truth value in the last column is correct or incorrect."
+      german "Entscheiden Sie für jede Tabellenzeile, ob der Wahrheitswert in der letzten Spalte korrekt oder fehlerhaft ist."
```

Comment:
Mir fehlen hier jetzt die Begriffe "Wahrheitstafel" und "truth table" (damit Studierende das in den Folien "nachschlagen" können).